src/apt_package_function/__init__.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""Tooling for creating apt repositories in Azure."""
+
+
+import logging
+import sys
+from datetime import datetime
+from pathlib import Path
+from typing import TextIO
+
+
+def common_logging(name: str, filename: str, stream: TextIO = sys.stdout) -> None:
+    """Set up common logging."""
+    if name == "__main__":
+        log_filename = Path(filename).with_suffix("").name
+    else:
+        log_filename = name
+
+    formatter = logging.Formatter("%(asctime)s %(levelname)-5.5s %(message)s")
+
+    log_path = Path(log_filename)
+
+    # Get the current time as a timestamp
+    timestamp = datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+
+    sojournerdir = Path(__file__).parent.parent.parent
```

Comment:
why is this called sojournerdir?

Reply:
Good question, should be rootdir really.